cdc/sinkv2/tablesink/progress_tracker.go

```diff
@@ -15,10 +15,11 @@ package tablesink
 
 import (
 	"context"
+	"math"
 	"sync"
+	stdAtomic "sync/atomic"
```

Comment:
```suggestion
	stdatomic "sync/atomic" or satomic "sync/atomic" 
```
We seem to use very little camel naming in package names.